lib/Transport.js

```diff
@@ -343,7 +345,7 @@ module.exports = class Transport
     {
       try
       {
-        data = String.fromCharCode.apply(null, new Uint8Array(data));
+        data = this.textDecoder.decode(data);
```

Comment:
Why is this just here for WebSocket binary messages?

Reply:
ok, I assume this is just a problem in binary WebSocket messages.

Reply:
Well, yes, maybe the summary of the PR is not entirely accurate.
(actually i had a 2 bytes long char in the SDP, not in a SIP header)

Reply:
Yes, but we do the conversion of binary to text in the whole WebSocket binary message, before we can parse SIP headers and body. So it's ok as it is.